rincon/roster.json

```diff
@@ -66,5 +66,10 @@
       "name": "Hassan Naeem",
       "github": "HassanEng12345"
      }
+  ],
+  "materials":[
+    {
+      "type2": "cv"
+    }
```

Comment:
I appreciate the enthusiasm, but it's a bit early for that :)

Reply:
Oh, sorry about this :)